app/codec.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package app;
+
+import "github.com/iov-one/weave/x/cash/codec.proto";
+import "github.com/iov-one/weave/x/sigs/codec.proto";
+import "gogoproto/gogo.proto";
+import "x/orderbook/codec.proto";
+
+// Tx contains the message
+message Tx {
+  // fee info, autogenerates GetFees()
+  cash.FeeInfo fees = 20;
+  repeated sigs.StdSignature signatures = 21;
+  // ID of a multisig contract.
+  repeated bytes multisig = 4;
+  // msg is a sum type over all allowed messages on this chain.
+  oneof sum {
+    cash.SendMsg send_msg = 51;
+    // space here to allow many more....
+
+    orderbook.CreateOrderBookMsg create_orderbook_msg = 100;
+    orderbook.CreateOrderMsg create_order_msg = 101;
+    orderbook.CancelOrderMsg cancel_order_msg = 102;
```

Comment:
We have updated recently the naming in `bnsd` to [always have the extension prefix](https://github.com/iov-one/weave/blob/master/cmd/bnsd/app/codec.proto#L36-L62). It is not a requirement. We did this to increase the readability and avoid possible name conflicts.

Reply:
I see. Will change it right away.